<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.exceptions.PropertyExceptionSpec" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-05-04T09:04:56" hostname="testing-docker-ace0605b-7337-48a1-8fdb-4d5d4cfd489f" time="0.006">
  <properties/>
  <testcase name="Exception should have proper formatted message[0]" classname="griffon.exceptions.PropertyExceptionSpec" time="0.003"/>
  <testcase name="Exception should have proper formatted message[1]" classname="griffon.exceptions.PropertyExceptionSpec" time="0.002"/>
  <testcase name="Exception should have proper formatted message[2]" classname="griffon.exceptions.PropertyExceptionSpec" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
